pi4j-core/src/main/java/com/pi4j/runtime/impl/DefaultRuntime.java

```diff
@@ -215,54 +217,68 @@ public Runtime initialize() throws InitializeException {
             plugins.clear();
 
             // container sets for providers and platforms to load
-            Set<Provider> providers = Collections.synchronizedSet(new HashSet<>());
-            Set<Platform> platforms = Collections.synchronizedSet(new HashSet<>());
+            Set<Platform> platforms = new HashSet<>();
```

Comment:
out of curiosity, why isn't synchronizedSet not needed here anymore?

Reply:
It was never necessary. Those are local variables, passed to the relevant classes, which then just copy/iterate the lists anyhow.